nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenTCPRecord.java

```diff
@@ -87,14 +93,28 @@
         "If the read times out, or if any other error is encountered when reading, the connection will be closed, and any records " +
         "read up to that point will be handled according to the configured Read Error Strategy (Discard or Transfer). In cases where " +
         "clients are keeping a connection open, the concurrent tasks for the processor should be adjusted to match the Max Number of " +
-        "TCP Connections allowed, so that there is a task processing each connection.")
+        "TCP Connections allowed, so that there is a task processing each connection. " +
+        "The processor can be configured to use an SSL Context Service to only allow secure connections. " +
+        "When a client connects to the processor using secure connection, the Distinguished Names of the client certificate's " +
+        "issuer and subject are added to the outgoing FlowFiles as attributes. " +
+        "The processor does not perform authorization based on Distinguished Name values, but since these values " +
+        "are attached to the outgoing FlowFiles, authorization can be implemented in the downstream flow using a RouteOnAttribute processor.")
```

Comment:
Recommend leaving off the specific processor reference and making it more generic:
```suggestion
        "are attached to the outgoing FlowFiles, authorization can be implemented based on these attributes.")
```